indico/queries/metrics.py

```diff
@@ -0,0 +1,102 @@
+from indico.client.request import GraphQLRequest
```

Comment:
I think this probably should be a submodule of model_groups to make it clear what these are.

Reply:
done.